verify.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+
+CLI="faas-cli"
+func_name="validate"
```

Comment:
I wasnt what this was and it took me a few minutes to figure out, might be nice to have a comment explaining `func_name`